src/services/phases/polkadot/xcm/index.ts

```diff
@@ -0,0 +1,91 @@
+import { SubmittableExtrinsic } from '@polkadot/api-base/types';
+import { parseEventXcmSent, XcmSentEvent } from '../eventParsers';
+import { WalletAccount } from '@talismn/connect-wallets';
+import { ISubmittableResult, Signer } from '@polkadot/types/types';
+
+export const submitUnsignedXcm = async (
```

Comment:
Nit: Let's rename to 
```suggestion
export const signAndSubmitXcm = async (
```

We don't submit an unsigned XCM message here.

Reply:
I see now it can be a confusing naming. Unsigned was supposed to mean that the transaction has not been yet signed. I'll rename.